substratevm/JFR.md

```diff
@@ -0,0 +1,79 @@
+This page covers how to use JDK Flight Recorder (JFR) with Java native images
+
+## What is JFR
+
+JFR is a production time profiling system built into OpenJDK. It is now supported in SubstrateVM so Java native images can continue to make use of the existing public Java API (jdk.jfr.*) and JFR itself, with a similar experience to using JFR in virtual machine mode.
```

Comment:
- JFR is not specific to OpenJDK.
- We don't use the term SubstrateVM in public documentation (use "Native Image" instead).
- I would replace "... in virtual machine mode" with "... in the Java HotSpot VM" (Native Image is also a VM).

Reply:
@jiekang, thanks for providing this comprehensive doc! We have set some guidelines when referring to GraalVM technologies, which might be helpful here:
- in reference to a runtime mode, use “on the JVM” but use"in Native Image." For example, “To run on the JVM or in Native Image." 
- in reference to the `native-image` utility, use “the Native Image builder.” Do not use “the Native Image generator.” 
- in reference to a build time, use “the image build time” or “building an image.”